src/modules/core/components/items-list-empty-illustration-placeholder/items-list-empty-illustration-placeholder.tsx

```diff
@@ -0,0 +1,31 @@
+import { type FC } from 'react';
+import clsx from 'clsx';
+// components
+import {
+  Illustration,
+  Placeholder,
+  Typography,
+} from '@/modules/core/components';
+
+import type { ItemsListEmptyIllustrationPlaceholderProps } from './items-list-empty-illustration-placeholder.interface';
+import styles from './items-list-empty-illustration-placeholder.module.scss';
+
+export const ItemsListEmptyIllustrationPlaceholder: FC<
```

Comment:
Can't we just add some variant to the Placeholder component instead of creating new component? In my opinion we have quite a lot of abstractions here that we might merge into one.

Reply:
Good idea. Will make it

Reply:
done